jujuclient/file.go

```diff
@@ -518,10 +515,9 @@ func (s *store) RemoveModel(controllerName, modelName string) error {
 	))
 }
 
-func updateModels(
-	controllerName string,
-	update func(*ControllerModels) (bool, error),
-) error {
+type modelUpdaterF func(storedModels *ControllerModels) (bool, error)
```

Comment:
conventionally it's "Func", not "F"
I think a more typical naming would be "updateModelFunc"